tests/test_eload_utils.py

```diff
@@ -0,0 +1,12 @@
+from unittest import TestCase
+
+from eva_submission.eload_utils import compare_sample_sets
+
+
+class TestEloadUtils(TestCase):
+
+    def test_compare_sample_sets(self):
+        assert compare_sample_sets([['S1', 'S2', 'S3'], ['S1', 'S2', 'S3']]) == 'single set'
+        assert compare_sample_sets([['S1'], ['S2'], ['S3']]) == 'unique sample sets'
+        assert compare_sample_sets([['S1'], ['S2'], ['S3', 'S4']]) == 'unique sample sets'
+        assert compare_sample_sets([['S1'], ['S2'], ['S3', 'S1']]) == 'overlapping sample names'
```

Comment:
i think you're missing the `unsorted single set` option...

(this is a helpful test btw, helps my small brain understand the logic here :stuck_out_tongue:)

Reply:
Thank added the extra test